src/translations/en.json

```diff
@@ -6374,11 +6374,20 @@
         "refresh_tokens": {
           "header": "Refresh tokens",
           "description": "Each refresh token represents a login session. Refresh tokens will be automatically removed when you click log out. Unused refresh tokens will be automatically removed after 90 days. The following refresh tokens are currently active for your account.",
-          "token_title": "Refresh token for {clientId}",
+          "ios_app": "iOS app",
+          "android_app": "Android app",
+          "current_session": "Your current session",
           "created_at": "Created {date}",
           "last_used": "Last used {date} from {location}",
           "not_used": "Has never been used",
-          "confirm_delete": "Are you sure you want to delete the refresh token for {name}?",
+          "expire_after_90_days": "Expires after 90 days of inactivity",
+          "never_expire": "Never expires",
+          "remove_expiration": "Remove expiration",
+          "confirm_remove_expiration_title": "Remove expiration date?",
+          "confirm_remove_expiration_text": "The refresh token for ''{name}'' will never expire. Removing the expiration date can reduce the security. Only disable expiration date on refresh token on devices which are rarely used (+90 days between usage).",
+          "remove_expiration_failed": "Failed to remove refresh token expiration",
+          "confirm_delete_title": "Delete refresh token?",
+          "confirm_delete_text": "The refresh token for ''{name}'' will be permanently deleted. The associated device will be disconnected.",
```

Comment:
```suggestion
          "confirm_delete_text": "The refresh token for ''{name}'' will be permanently deleted. This will end the login session on the associated device.",
```